ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/prepare-and-run-synthea.py

```diff
@@ -78,6 +82,18 @@ def validate_and_run(args):
         sys.exit(1)
     
     end_state_properties_file = read_file_lines(end_state_file_path)
+    
+    ## If contract target is requested via arg5, add a line to replace the synthea properties:
+    ## exporter.bfd.partd_contract_start and exporter.bfd.partd_contract_count
```

Comment:
Not sure it's necessary to call out the specific argument number here.  Calling out `arg5` specifically in the comment will require someone to scan and change comments if we refactor arguments later.

Reply:
Good point, fixed to just use the param name.